Formula/neovim-qt.rb

```diff
@@ -0,0 +1,32 @@
+class NeovimQt < Formula
+  desc "Neovim GUI, in Qt5"
+  homepage "https://github.com/equalsraf/neovim-qt"
+  url "https://github.com/equalsraf/neovim-qt/archive/v0.2.16.1.tar.gz"
+  sha256 "971d4597b40df2756b313afe1996f07915643e8bf10efe416b64cc337e4faf2a"
+  license "ISC"
+  head "https://github.com/equalsraf/neovim-qt.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "neovim"
+  depends_on "qt@5"
+
+  def install
+    mkdir "build" do
+      system "cmake", "-DCMAKE_BUILD_TYPE=Release", ".."
+      system "make"
+
+      if OS.mac?
+        prefix.install "bin/nvim-qt.app"
+        bin.install_symlink prefix/"nvim-qt.app/Contents/MacOS/nvim-qt"
+      end
+
+      if OS.linux?
+        bin.install "bin/nvim-qt"
+      end
+    end
+  end
+
+  test do
+    system bin/"nvim-qt", "--help"
```

Comment:
Could we start an `nvim-qt` process, and then use `neovim-remote` to try to interact with it? That way we get to test a bit more than `--help`, which we strongly discourage for tests.

Reply:
if it avoid some dependencies, you could also use just `nvim` instead of `neovim-remote` to interact

Reply:
We could declare the dependency as test-only, so it should have no user-facing impact. Though if we do it directly with `nvim` that's fine too.

Reply:
Would using `neovim-remote` also prevent the test failure I reported in the OP?

Reply:
It might; I'm not sure. We can try it out in CI to see.

Reply:
I realized that you don't need Neovim-remote to interact with Neovim. Recent versions of Neovim-Qt can pass arbitrary arguments along to the `nvim` program.

I copied the `test` block from the Neovim formula. It appears to work, but I still get the "Killing child processes"/"terminated by uncaught signal" issue. I wonder if this is a false error? Maybe Neovim-Qt sends SIGTERM to the underlying Neovim process?

Neovim-Qt has the ability to connect to an existing Neovim process with `nvim-qt --server`, which might avoid the SIGTERM problem. But then it doesn't support passing arguments to the underlying Neovim process!